File: csharp/Command/StockController.cs
namespace Command;

public class StockController
{
    List<IOrder> _orders = new List<IOrder>();

    public void TakeOrder(IOrder order)
    {
        _orders.Add(order);
    }

    public void PlaceOrders()
    {
        foreach (IOrder order in _orders)
        {
            order.Execute();
        }

        _orders.Clear();
    }
}

File: java/command/src/main/StockController.java
package main;

import java.util.ArrayList;
import java.util.List;

public class StockController {

    List<Order> orders = new ArrayList<>();

    public void TakeOrder(Order order)
    {
        orders.add(order);
    }

    public void PlaceOrders()
    {
        for (Order order : orders) {
            order.Execute();
        }

        orders.clear();
    }
}